api/src/endpoints/svgThumbnail.js

```diff
@@ -7,10 +7,25 @@ const VALID_MODELS = models.map((m) => m.short_name);
 
 const routes = express.Router();
 
+function validateDimension(dimensionString) {
+  const minDim = 100;
+  const maxDim = 600;
+  if(dimensionString){
+    if (!/^\d+$/.test(dimensionString)) {
+      throw new Error(`Invalid dimension provided: ${dimensionString}. The width and/or height parameters should be an integer between ${minDim} and ${maxDim}.`);
```

Comment:
Sure, I'm up for reconsidering.
The `and/or` is meant to indicate that providing only one of the params is also acceptable.

Reply:
Sorry I should have clarified, I feel like we should go with one of these options:
- use `or` since it only uses `width` or `height` depending on which is less
- use `and/or` but change the resize logic to something like `fill` so that it uses both `width` and `height`

Reply:
[`fill`](https://sharp.pixelplumbing.com/api-resize#resize) cannot be used because it changes the aspect ratio. I went for `inside`, but I guess `contain` and `cover` are equally good. We could point to this documentation in the error message.

Reply:
@nanjiangshu based on your comment I would guess you're advocating for `'contain'`.

Reply:
Great! That leaves us only `cover`.